From 75fe0fbe254f63fe58d22f2d6090e9a8b3a6fe41 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Tue, 5 Mar 2024 12:03:23 +0000
Subject: [PATCH] yaffs_guts.h: define YTIME_T if not already defined

Fixed:
yaffs_guts.h:501:9: error: unknown type name 'YTIME_T'
  501 |         YTIME_T yst_uid;

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 core/yaffs_guts.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/yaffs_guts.h b/core/yaffs_guts.h
index 4bd2646..507bf16 100644
--- a/core/yaffs_guts.h
+++ b/core/yaffs_guts.h
@@ -491,6 +491,10 @@ struct yaffs_obj {
 
 	YCHAR short_name[YAFFS_SHORT_NAME_LENGTH + 1];
 
+#ifndef YTIME_T
+#define YTIME_T     time_t
+#endif
+
 #ifdef CONFIG_YAFFS_WINCE
 	//these are always 64 bits
 	u32 win_ctime[2];
-- 
2.34.1

